### ISSUE TITLE
Build error with version 1.0.7

### ISSUE THREAD

**Issue by DanielOuwendijk** [AzureAD/microsoft-authentication-library-for-objc#870]
Hello,

I have just updated my MSAL package from 1.0.3. to 1.0.7. When I run my project with 1.07, it gives me the following error:

/Users/me/Desktop/project Name/Pods/MSAL/MSAL/IdentityCore/IdentityCore/src/requests/sdk/msal/MSIDDefaultSilentTokenRequest 2.m:121:50: No visible @interface for 'MSIDDefaultTokenCacheAccessor' declares the selector 'getAccountForIdentifier:authority:context:error:'

<img width="259" alt="Screenshot 2020-03-13 at 13 21 33" src="https://user-images.githubusercontent.com/50083609/76620832-5d832c00-652e-11ea-93a4-948ffd2cdae9.png">

<img width="876" alt="Screenshot 2020-03-13 at 13 21 45" src="https://user-images.githubusercontent.com/50083609/76620798-4d6b4c80-652e-11ea-933e-50a60d418a69.png">

Any idea of why this is happening? I don't see anyone else having this problem. Am I missing something?

**Comment by rohitnarula7176** [comment]
@DanielOuwendijk Hey Daniel, when you clone the repo from git, you need to run git submodule update. If you download the package from releases, then you need to look up the project
microsoft-authentication-library-for-objc-1.0.7.zip.

**Comment by DanielOuwendijk** [comment]
I use MSAL from cocoapods. Do you know what command I need to run to solve this? (I haven't researched this, so I'm going to check now)

**Comment by antrix1989** [comment]
Could you try to run `pod update` and see if it helps? You could have cached pods on your machine. Please let me know if it works.

**Comment by DanielOuwendijk** [comment]
Okay, this is weird.

Pod update solved it! but I tried this previously...? My story goes:

1. In de pod file i removed MSAL 1.0.3. and changed it to 1.0.7. with `pod update`
2. I build the project and it gave the error.
3. In de pod file I changed MSAL 1.0.7. back to 1.0.3. so it would at least work again
4. I asked this question on github
5. You say try `pod update`
6. I think: "sure lets try it again"
7. Now it suddenly does work!?

Well then, thank you!
